### PR TITLE
Check changes.

### PR DESCRIPTION
**Pull request by sonnle**
I know that we discussed a lot of different things that will need to be changed that are minor. (Naming of variables, State changes, Speaker implementation, etc.) I made some of those changes, but take a look to see if there is anything else we should push before we go ahead and branch off master for our new stuff.

### CODE REVIEW

File: Speaker/Speaker.h [hunk 1]
```diff
@@ -25,20 +25,21 @@ void Speaker_Initiate(void);
  *  Parameter loops dictates how long the Speaker is on for
  *
  *  @param int: loops - number of loops to play sound
+ *	@param int: frequency - time in between oscillations
```
Comment (mfeist) [inline Speaker/Speaker.h]: Will there be this extra tab in the code or is it just on github?

**Comment by mfeist** [comment]
Other than that, it looks good.

**Comment by sonnle** [comment]
I feel like we may be missing something that was discussed. Can you think of anything else that should be changed?

**Comment by mfeist** [comment]
O, you mentioned a function to do:
Control_GlobalStatus = None_Status;
Control_GlobalState = Undefined_State;
StepperMotor_CurrentBasalDose = 0;
StepperMotor_CurrentBolusDose = 0;
Since those exact lines are used in the StepperMotor_StepForward as well as in the Main loop.

**Comment by sonnle** [comment]
Any recommendations for function name? I was thinking Control_StateStatusReset() or Control_DosageReset()

**Comment by mfeist** [comment]
I like Control_DosageReset()

**Pull request merged**